game/txt/Makefile

```diff
@@ -17,9 +17,15 @@ TXT=help.txt news.txt
 # below.)
 INDEX_FLAGS=
 
+CHANGES_FILES=$(wildcard ../../CHANGES*)
+HELP_CHANGES_FILES=$(patsubst ../../CHANGES.%,hlp/pennv%.hlp,$(CHANGES_FILES))
+
 all: $(TXT)
 
-help.txt: hlp/*.hlp hlp compose.sh
+$(HELP_CHANGES_FILES): hlp/pennv%.hlp: ../../CHANGES.%
+	../../utils/mkvershlp.pl hlp ../../CHANGES*
```

Comment:
Note that we're calling `mkvershlp.pl` with ALL of the changes files anytime any one of them changes. That's intentional, since the script needs to be given all of the changes files at once to determine what the newest patchlevel is, to be listed in `help changes`.